-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

- From e0b8bf776349659d5814f237e7f744ab08e3be64 Mon Sep 17 00:00:00 2001
From: Sebastian Bach <sebtomba@gmail.com>
Date: Mon, 15 May 2023 15:37:42 +0200
Subject: [PATCH] [PM-6116] Remove commented out code

- ---
 pallets/template/src/ledger_bridge.rs | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/pallets/template/src/ledger_bridge.rs b/pallets/template/src/ledger_bridge.rs
index 0f5dd6d4..d25d5d7d 100644
- --- a/pallets/template/src/ledger_bridge.rs
+++ b/pallets/template/src/ledger_bridge.rs
@@ -10,17 +10,10 @@ use coin_structure::contract::Address as ContractAddress;
 use midnight_base_crypto::hash::{HashOutput};
 
 pub enum ValidationError {
- -	// DeserializationError(io::Error),
 	DeserializationError(SerdeError),
 	MalformedError(ledger::error::MalformedTransaction)
 }
 
- -// impl From<io::Error> for ValidationError {
- -// 	fn from(value: io::Error) -> Self {
- -// 		ValidationError::DeserializationError(value)
- -// 	}
- -// }
- -
 impl From<SerdeError> for ValidationError {
 	fn from(value: SerdeError) -> Self {
 		ValidationError::DeserializationError(value)
@@ -54,7 +47,6 @@ pub fn init_new_state() -> Vec<u8> {
 }
 
 fn deserialize_transaction(encoded_transaction: &Vec<u8>) -> Result<Transaction<Proof>, SerdeError> {
- -	// ledger::serialize::deserialize(encoded_transaction.as_slice())
 	let tx: &str = std::str::from_utf8(encoded_transaction).unwrap();
 	let tx: Transaction<Proof> = from_str(tx)?;
 	Ok(tx)
- -- 
2.46.0

-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEE9Oz16eF+Nv5C7TpSdEmbFI1hGmwFAmhtArMACgkQdEmbFI1h
Gmyr1A//Y0K7JGTdTDHJh5A8N4PbNHb4fPQxjpho3tcbkmwA8qbdAdcf/u4Aqpcp
+LGAgcsbn3ofhImIg1WraoznpfdfUKcVYI/J9CU0tJSPv8prdEeb/ShYMu0oSbUD
yhLzxuz6Zmqi1XSn1dXYCWe5tmttf7DfMSrm5Z7Rl10dO92DGGP0ATeqKdH2B88Q
EE7VhqzCP21m4Km93b4HbCZL8fcYfkvum8uCHiKaXDwKFM7/knklgD5IK9CuTZSG
VzBZinfktzG3RTYLe8UduqtBzkwebSkZTXhrEFIPOd4PQ01SfoHhyOACk32u3XIA
HeooZrn6SycCbWWlZ1IqrX3BjJv37dGi9k2pbaopeYhE6CulTTh9B7vKigQI/lci
AfZ6NwZsUxgC3Fov1b8bESet9tRuYWqRTJ8oLQcOci3ObiWDkjeyrgWCtQ+nWUnN
8Wm+sJR4/3rY8GVsIxSchkSU8gTru8yGW2l6hAYRnT3yA5C+/SoK2Yz/48iIpbUF
Dil2v4co4o8p1KamfdfEBsAMt46I9iCf0ywHlQymVo7Nz8UNeKOQ4jEVuJl94bQX
cx9FOVElfE+YPv2kUJr/iRn65yCjtRZTi2LVOgyWeNJF8SmOfyq3l8hc0abdN3WM
ImGZqk5Aqkr+WKF7/sxavYDkIdA3O2ynMXA++5tEztlQb7XMLR0=
=Bk0e
-----END PGP SIGNATURE-----
